proposals/nnnn-structured-concurrency.md

```diff
@@ -138,11 +140,15 @@ The result of `runDetached` is a task handle, which can be used to retrieve the
 
 ### Tasks
 
-A task can be in one of three states:
+A task can be in one of three (**TODO**: four?) states:
```

Comment:
not really nitpicking but trying to understand myself the states we envision a task to be in... Maybe I am nitpicking after all though? 🤔